src/main/java/edu/gatech/chai/VRDR/messaging/DemographicsCodingUpdateMessage.java

```diff
@@ -40,8 +40,8 @@ public void setDemographicCodedContentBundle(DemographicCodedContentBundle demog
 
     public DemographicsCodingUpdateMessage(BaseMessage messageToCode) {
         this(messageToCode == null ? null : messageToCode.getMessageHeaderId(),
-                messageToCode == null ? null : messageToCode.getMessageDestination(),
-                messageToCode == null ? null : messageToCode.getMessageSource());
+		messageToCode == null ? null : messageToCode.getMessageSource(),
```

Comment:
Don’t flip this one it’s meant to be this way

Reply:
I reverted the changes, and found out that the example files in question all lack the "source/endpoint" attribute.